docs/repo-config.csv

```diff
@@ -1,5 +1,5 @@
-Repository's Location,Branch,File formats,Ignore Glob List,Ignore standalone config,Ignore Commits List
-https://github.com/reposense/testrepo-Alpha.git,master,,,,2fb6b9b2dd9fa40bf0f9815da2cb0ae8731436c7;c5a6dc774e22099cd9ddeb0faff1e75f9cf4f151;cd7f610e0becbdf331d5231887d8010a689f87c7;768015345e70f06add2a8b7d1f901dc07bf70582
-https://github.com/reposense/testrepo-Beta.git,master,java;adocs,docs**,yes,
-https://github.com/reposense/testrepo-Delta.git,master,html;css,,,
-https://github.com/reposense/RepoSense.git,master,,,,
+Repository's Location,Branch,File formats,Ignore Glob List,Ignore standalone config,Ignore Commits List,Ignore Authors List
+https://github.com/reposense/testrepo-Alpha.git,master,,,,2fb6b9b2dd9fa40bf0f9815da2cb0ae8731436c7;c5a6dc774e22099cd9ddeb0faff1e75f9cf4f151;cd7f610e0becbdf331d5231887d8010a689f87c7;768015345e70f06add2a8b7d1f901dc07bf70582,
```

Comment:
This is a sample `.csv` so do add an example here for the ignore author list too
E.g. `Author Name One;Author Name Two`